app/screens/channel_info/edit_channel/edit_channel.tsx

```diff
@@ -12,6 +12,7 @@ import {t} from '@utils/i18n';
 import {preventDoubleTap} from '@utils/tap';
 
 interface EditChannelProps {
+    testID: string;
```

Comment:
No component should require the `testID` as mandatory

```suggestion
    testID?: string;
```